force-app/main/default/lwc/accountModelSettings/accountModelSettings.js

```diff
@@ -14,8 +25,56 @@ export default class AccountModelSettings extends LightningElement {
 
     labelReference = {
         stgAccountModelSettingsTitle,
+        stgAccModelTitle,
+        stgHelpAccountModel,
+        stgAdminAccountRecordType,
+        stgHelpAdminRecType,
+        stgAccountRecordTypeSupportsHHAddress,
+        stgHelpHouseholdRecType,
     };
 
+    @wire(getAccountModelSettingsViewModel)
+    accountModelSettingsViewModel({ error, data }) {
+        if (data) {
+            this.accountModelSettingsViewModel = data;
+        } else if (error) {
+            console.log("error retrieving accountmodelsettingsvmodel");
```

Comment:
Please comment this out for now. Error handling for display will need new mocks and a different pass.